contracts/core/tokens/rebalancing-libraries/StandardStartRebalanceLibrary.sol

```diff
@@ -111,6 +111,13 @@ library StandardStartRebalanceLibrary {
             _vaultAddress
         );
 
+        // Require remainingCurrentSets to be greater than minimumBid otherwise no bidding would
+        // be allowed
+        require(
+            biddingParameters.remainingCurrentSets >= biddingParameters.minimumBid,
```

Comment:
Can we also have a function in settleLibrary to reflect this equality? It's also in `endFailedAuction`